OPERATING.md

```diff
@@ -0,0 +1,79 @@
+# Operating a Convene Neighborhood
+
+- [Social Responsibilities](#social-responsibilities)
+  - [As a Cooperative](#as-a-cooperative)
+  - [As a Person](#as-a-person)
+  - [As a Legacy Organization](#as-a-legacy-organization)
+- [Economic Model](#economic-model)
+  - [As a Cooperative](#as-a-cooperative-1)
+  - [As a Person](#as-a-person-1)
+  - [As a Classic Corporation](#as-a-classic-corporation)
+- [Technology Details](#technology-details)
+  - [As a Cooperative](#as-a-cooperative-2)
+  - [As a Person](#as-a-person-2)
+  - [As a Classic Non-Profit, Government, or Corporation](#as-a-classic-non-profit-government-or-corporation)
+
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
+
+Convene is deployed as a `Neighborhood` on commodity cloud infrastructure. It allows an `Operator`, (either a person or a worker and community-owned mutual-benefit technology cooperative) to support the digital operations of mutual-aid and mutual-benefit organizations (or `Neighbor`s).
+
+Where `Contributors` labor to research, design, test, and build software that delights `Neighbor`s and grants them superpowers, `Operators` labor to distribute this software to `People`. `Operators`:
+
+- Nurture the `Neighborhood` by setting and maintaining healthy boundaries between `People`, `Neighbor`s and other `Neighborhood`s.
+- Welcome `People` to the `Neighborhood` and support them in becoming a good `Neighbor`.
+- Help `Neighbor`s get the most from their `Neighborhood` through carework, gluework and technical support.
+
+This guide covers the social, economic, and technological aspects of operating a Convene `Neighborhood`. Each section starts by describing the shared context, and then splitting between the three kinds of `Operator`s
+
+- Worker-owned and Directed Cooperative, Mutual-Benefit, or Mutual Aid Organizations
+- Individual People
+- Legacy Organizations, such as traditional Non-profits, Governments, or Corporations
+
+## Social Responsibilities
```

Comment:
Are these guidelines for how to use convene in a Socially responsible manner?

We can't enforce/control how people run their orgs. But the least we can do is make it clear where we stand and provide education and guidance.